ssl/tls13_client.cc

```diff
@@ -889,6 +890,37 @@ static enum ssl_hs_wait_t do_send_client_certificate_verify(SSL_HANDSHAKE *hs) {
   return ssl_hs_error;
 }
 
+static enum ssl_hs_wait_t do_process_pha(SSL_HANDSHAKE *hs) {
```

Comment:
I'm just really nitpicking here.. But since the handshake states flow down in the file, it would nice to see `do_process_pha` defined after `do_complete_second_flight`, and see that `hs->tls13_state = state_done;` is the last one at the bottom.